src/reformat_libyuv.c

```diff
@@ -37,6 +37,7 @@ unsigned int avifLibYUVVersion(void)
 #if defined(__clang__)
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wstrict-prototypes" // "this function declaration is not a prototype"
+#pragma clang diagnostic ignored "-Wnewline-eof" // "no newline at end of file"
```

Comment:
Yuan: Could you add a newline character at the end of libyuv.h?

Reply:
The newline was removed (probably by mistake) in libyuv commit 48d167108ff60850ef788d25ef328d223a3f25c4 (version 1792, commited at 2021-09-25). I can write a CL to add it back, but there's many affected versions. Should I remove this line as well?